stores/appStore.ts

```diff
@@ -0,0 +1,21 @@
+import { defineStore } from "pinia";
+import type { AppState } from "@/types/storeTypes";
+
+export const useAppStore = defineStore("app", {
+  state: (): AppState => ({
+    largeScreen: true,
+  }),
+  getters: {
+    screenIsLarge: (state: AppState): boolean => {
+      return state.largeScreen;
+    },
+  },
+  actions: {
+    initializeAppState(): void {
+      this.largeScreen = true;
+    },
+    setScreenSize(bool: boolean): void {
```

Comment:
should this be `setScreenSizeToLarge` .. also function param should be descrptive .. eg. isLargeScreen rather than bool

Reply:
Good points.

Reply:
I have changed the function param to be called large, otherwise kept this the same